recipes/rocm-cmake/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "rocm-cmake" %}
+{% set version = "2.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RadeonOpenCompute/rocm-cmake/archive/roc-{{ version }}.tar.gz
+  sha256: 73ec6e1d248902916062bc186b5c937f3387f8eab54ff3314030b53cc0a92d46
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+  run:
```

Comment:
Are these empty sections required?

Reply:
Technically not here. This is just to make sure the build tools are in `build` instead of `host`. conda-build has a habit of merging `build` and `host` if there's only `build` (for conda-build 2.x compat)